client/src/test/java/io/pravega/schemaregistry/client/SchemaRegistryClientConfigTest.java

```diff
@@ -46,9 +47,19 @@ public void testAuthConfig() {
         SchemaRegistryClientConfig config = SchemaRegistryClientConfig.builder().build();
         assertFalse(config.isAuthEnabled());
 
-        config = SchemaRegistryClientConfig.builder().authentication("method", "token").build();
+        config = SchemaRegistryClientConfig.builder().authentication(new CredentialProvider() {
+            @Override
+            public String getMethod() {
+                return "method";
+            }
+
+            @Override
+            public String getToken() {
+                return "token";
+            }
+        }).build();
         assertTrue(config.isAuthEnabled());
-        assertEquals(config.getAuthMethod(), "method");
-        assertEquals(config.getAuthToken(), "token");
+        assertEquals(config.getCredentialProvider().getMethod(), "method");
```

Comment:
The left element of the method should be the `expected`, and the right the actual. If this test failed the error'd mislead.